### PR TITLE
clean up vaults strings, add new ones

### PR DESCRIPTION
**Pull request by tyleroooo**

### CODE REVIEW

File: config/localization/en/app.json [hunk 1]
```diff
@@ -1526,31 +1526,20 @@
...
       "VAULT": "Vault",
-      "ASSET_VAULT": "{ASSET} Vault",
-      "VAULTS": "Vaults",
-      "VAULT_DETAILS": "Vault Details",
-      "VAULT_FAQS": "Vault FAQs",
-      "VAULT_BALANCE": "Vault Balance",
-      "VAULT_POSITION": "Vault Position",
-      "VAULT_LEVERAGE": "Vault Leverage",
-      "VAULT_ALL_TIME_PNL": "All-time Vault P&L",
+      "YOUR_ALL_TIME_PNL": "Your All-time P&L",
```
Comment (moo-onthelawn) [inline config/localization/en/app.json:1530]: nit: `All-Time`?

File: config/localization/en/app.json [hunk 2]
```diff
@@ -1526,31 +1526,20 @@
...
       "VAULT_EQUITY": "Vault Equity",
       "VAULT_PNL": "Vault P&L",
-      "DEPOSITS_AND_WITHDRAWALS": "Deposits & Withdrawals",
-      "VIEW_ALL_VAULTS": "View All Vaults",
-      "YOU_HAVE_NO_VAULT_DEPOSITS": "You have no vault deposits."
+      "YOUR_DEPOSITS_AND_WITHDRAWALS": "Your Deposits & Withdrawals",
+      "YOU_HAVE_NO_VAULT_DEPOSITS": "You have no vault deposits.",
+      "7_DAYS": "7d",
+      "30_DAYS": "30d",
+      "90_DAYS": "90d",
```
Comment (moo-onthelawn) [inline config/localization/en/app.json:1541]: nit: unsure if we need to translate these strings, or if we can just use [getStringsForTimeInterval](https://github.com/dydxprotocol/v4-web/blob/main/src/lib/timeUtils.ts#L27)

Comment (moo-onthelawn) [reply config/localization/en/app.json:1541]: if we do need to translate, mild preference for something like [X_MINUTES_LOWERCASED](https://github.com/dydxprotocol/v4-localization/blob/main/config/localization/en/app.json#L35) which is more reusable

**Review by moo-onthelawn — APPROVED** [review]
curious if we've confirmed if any of these strings might be used on mobile?

File: config/localization/en/app.json (hunk 1 above)
Comment (tyleroooo) [reply config/localization/en/app.json:1530]: for some reason Dan has the t always lowercase in the designs.  I guess I'll leave it that way until someone complains

**Pull request merged**